lib/puppet/provider/vcsrepo/svn.rb

```diff
@@ -157,17 +156,16 @@ def includes=(desired)
     update_includes(new_paths)
   end
 
-
   private
 
   def get_includes(directory)
     at_path do
       args = buildargs.push('info', directory)
-      if svn(*args)[/^Depth:\s+(\w+)/m, 1] != 'empty'
+      if svn(*args)[%r{^Depth:\s+(\w+)}m, 1] != 'empty'
         return directory[2..-1].gsub(File::SEPARATOR, '/')
       end
       Dir.entries(directory).map { |entry|
-        next if entry == '.' or entry == '..' or entry == '.svn'
+        next if entry.include?('.') || entry.include?('..') || entry.include?('.svn')
```

Comment:
Bit wary of this, the functionality isn't really the same

== '.' for example would only match .
However .include?('.') would match on . but also on file.txt

Reply:
this is meant to be an array of the comparison values with the include being the entry